cabot_alert_hipchat/models.py

```diff
@@ -70,13 +71,21 @@ def _send_hipchat_alert(self, message, color='green', sender='Cabot'):
 
         room = env.get('HIPCHAT_ALERT_ROOM')
         api_key = env.get('HIPCHAT_API_KEY')
-        url = env.get('HIPCHAT_URL')
+        domain = env.get('HIPCHAT_DOMAIN', 'api.hipchat.com')
 
-        resp = requests.post(url + '?auth_token=' + api_key, data={
-            'room_id': room,
-            'from': sender[:15],
+        # Backwards compatibility
+        if env.get('HIPCHAT_URL'):
+            domain = urlparse(env.get('HIPCHAT_URL')).hostname
```

Comment:
So if both `HIPCHAT_DOMAIN` and `HIPCHAT_URL` are set, the latter will take precedence?